trpcultivate_phenotypes/trpcultivate_phenotypes.libraries.yml

```diff
@@ -0,0 +1,24 @@
+# Tripal Cultivate Phenotypes Libraries.
+
+# Script to auto-select field content for quick alteration of value.
+script-autoselect-field:
```

Comment:
Maybe we should prefix these with `tcp` or `trpcultivatepheno` instead of `script` to make their names more unique?